README.md

```diff
@@ -220,11 +220,11 @@ Task {
 import FluidAudio
 
 // Use the default configuration (already optimized for best results)
-let vadConfig = VadConfig()  // Threshold: 0.445, optimized settings
+let vadConfig = VadConfig()  // Threshold: 0.5, optimized settings
 
 // Or customize the configuration
```

Comment:
Lets jsut remove , the readme is growing a bit large
```suggestion
```

Reply:
I think we should still keep it, jsut a very simplified version with default values